### ISSUE TITLE
Difficulty in machu.treeplot

### ISSUE THREAD

**Issue by BeardLi** [wxguillo/machuruku#1]
Hello,
         I ran into an error when I use "machu.treeplot" in your package, when I was using totorial files.
         It said:
`machu.treeplot(trees, upperX = 3, timelabeloffset = 1.25, nodecirclecolor = "darkseagreen2")
Error in DataMask$new(.data, caller_env) :Missing parameter "caller_env", no default value`
          Looking forward to your reply!

**Comment by wxguillo** [comment]
Hi BeardLi, thanks for your comment. I ran through the tutorial and didn't find your exact error at this part, but a different one. There was an issue with the lapply function I was using to constructed the multi-tree plots. I fixed it by simply rewriting the code to use loops rather than lapply (ironically, similar to an earlier version). I believe this should fix your issue as well. Go ahead and reinstall the package and see if it works.
